FinniversKit/Sources/Components/Infobox/InfoboxSwiftUIView.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import SwiftUI
+
+public struct InfoboxSwiftUIView: View {
+    public let style: InfoboxView.Style
+    public var viewModel: InfoboxViewModel?
+    public var onPrimaryButtonTapped: (() -> Void)?
+    public var onSecondaryButtonTapped: (() -> Void)?
+
+    public init(
+        style: InfoboxView.Style,
+        viewModel: InfoboxViewModel? = nil,
```

Comment:
Is not having a view model really an option?

Reply:
Yeah I was surprised too but it seems so, here's the original in `InfoboxView`

```
public var model: InfoboxViewModel? {
        didSet {

        }
}
```

🤔 but I don't know if that is a side-effect or workaround for some compiler related maybe?

Reply:
My suggestion is to require the view model and see what problems crop up. 😄

Reply:
So far none...and it makes the code prettier so let's change it to that ✅